### PR TITLE
Network: added package tags in java readme

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

File: specification/network/resource-manager/readme.java.md [hunk 1]
```diff
@@ -38,6 +39,19 @@ regenerate-manager: true
...
+### Tag: package-2019-02 and java
+
+These settings apply only when `--tag=package-2019-02 --java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == 'package-2019-02' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.network.v2019_02_01
+  output-folder: $(azure-libraries-for-java-folder)/network/resource-manager/v2019_02_01
+regenerate-manager: true
```
Comment (kpajdzik) [inline specification/network/resource-manager/readme.java.md]: I think the formatting is incorrect.

**Review by kpajdzik — CHANGES_REQUESTED** [review]
Please fix YAML formatting

File: specification/network/resource-manager/readme.java.md (hunk 1 above)
Comment (iscai-msft) [reply specification/network/resource-manager/readme.java.md]: thanks for catching this

**Review by kpajdzik — APPROVED** [review]

**Comment by kpajdzik** [comment]
@iscai-msft, Do you want me to merge it?

**Pull request merged**